README.md

```diff
@@ -1,90 +1,130 @@
 # Flashbots
 
-Flashbots is a research and development organization formed to mitigate the negative externalities and existential risks posed by miner-extractable value (MEV) to smart-contract blockchains, starting with Ethereum. Our primary focus is to enable a *permissionless, transparent, and fair ecosystem* for MEV extraction. Flashbots arose out of the *MEV Pi-rate Ship*, a neutral, chain-agnostic, interdisciplinary research collective that supports MEV-related theoretical and empirical research. 
+Flashbots is a research and development organization working on mitigating the negative externalities of current MEV extraction techniques and avoiding the existential risks MEV could cause to state-rich blockchains like Ethereum. Our work falls under three efforts: Democratizing Access to MEV Revenue, Bringing Transparency to MEV Activity and Distributing MEV Revenue.
 
-Flashbots is committed to open source. The best way to learn about Flashbots is to follow our progress in each repo. Find out more about Flashbots organization in our [medium post](https://medium.com/flashbots/frontrunning-the-mev-crisis-40629a613752) _Flashbots: Frontrunning the MEV-crisis_.
+We've released two projects so far: 
+* MEV-Geth & MEV-Relay: a proof of concept communication channel between miners and trader for permisionless, transparent and efficient MEV extraction.
+* MEV-Explore: a public dashboard and live transactions explorer of MEV activity.
 
-Flashbots research and development are tightly-coupled dual engines that propel Flashbots in a phased approach:
-- **Long-term oriented research effort:** They spec out and update our roadmap, define our organization’s phases and identify key milestones associated with each of them;
-- **Milestone-oriented engineering effort:** They are organized as product-focused teams that ship core infrastructure and ecosystem tools, while collecting data and producing other artifacts that feed back into research.
+You can interact with Flashbots on Discord and Github. We have dedicated channels on Discord for each of our efforts and welcome you to contribute in the relevant channels. Our work is open source and you can follow our progress in each repository of the Flashbots organization. In addition, we hold several recurring community calls: a monhtly MEV Roast, a bi-weekly research workshop and a monthly transparency call to update the community on our progress. Subscribe to our public calendar [here](https://bit.ly/3uJAHpo).
```

Comment:
monhtly -> monthly

Reply:
I'd shorten this a bit, perhaps:
"We have dedicated channels on Discord for each of our efforts and welcome you to contribute in the relevant channels." -> "We have dedicated channels on Discord for each of our efforts and welcome contributions".

Reply:
The calendar part could be moved to the "resource list" below

Reply:
Nice! For calendar, I'd like to leave it at the top because it's a key info I'd like reader to access asap?